lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -409,6 +411,22 @@ function dosomething_user_validate_address_field($form, &$form_state) {
 }
 
 /**
+ * Validates that password is 6 or more characters long.  
+ *
+ * @param array $form
+ *  A drupal form.
+ * @param array $form_state
+ *  A drupal form_state array.
+ *
+ */
+function dosomething_user_validate_password_length($form, &$form_state) {
```

Comment:
We have it client-side [here](https://github.com/DoSomething/phoenix/blob/5896b2823f770c6aa3a1e9cf46d474ef4ae60564/lib/themes/dosomething/paraneue_dosomething/js/validators/auth.js#L215) but I couldn't find anything that checked it anywhere else.